eoscostarica/eosio-dashboard
# Fix/sidebar position 908

### Move the sidebar down from the header and up to the footer

### What does this PR do?

- Resolve #908
- Rearrange flex boxes to move the sidebar
- Add shadow to the sidebar

### Steps to test

1. Run the project locally
1. Check that the sidebar is below the header, above the footer and has a fixed position

### Screenshots

![imagen](https://user-images.githubusercontent.com/66583677/192655687-f158a2aa-d461-457d-9a3d-1cbe3b54799f.png)
![imagen](https://user-images.githubusercontent.com/66583677/192655703-88948386-2622-4e97-8b31-c49abc0f9107.png)
![imagen](https://user-images.githubusercontent.com/66583677/192655729-dc7ba76f-5466-41ae-bb0b-117952a68a8d.png)



#### CheckList
